diff --git a/packages/compositor-proxy/src/NativeWaylandClientSession.ts b/packages/compositor-proxy/src/NativeWaylandClientSession.ts
index 612d79c..2d7cf16 100644
--- a/packages/compositor-proxy/src/NativeWaylandClientSession.ts
+++ b/packages/compositor-proxy/src/NativeWaylandClientSession.ts
@@ -77,10 +77,11 @@ export function createNativeClientSession(
   )
 
   setClientDestroyedCallback(wlClient, () => {
+    logger.info(`setClientDestroyedCallback`)
     for (const destroyListener of nativeClientSession.destroyListeners) {
       destroyListener()
     }
-    nativeAppContext.sendClientConnectionsDisconnect()
+    //nativeAppContext.sendClientConnectionsDisconnect()
     nativeClientSession.destroyListeners = []
     nativeClientSession.destroyed = true
   })
